config/config.exs

```diff
@@ -37,7 +37,8 @@ config :wanda, WandaWeb.Endpoint,
   live_view: [signing_salt: "j6kcshS4"]
 
 config :cors_plug,
-  origin: [System.get_env("CORS_ORIGIN", "http://localhost:4000")]
+  origin: [System.get_env("CORS_ORIGIN", "http://localhost:4000")],
+  enabled: true
```

Comment:
maybe we could create a `config  :wanda, :plugs, cors_plug_enabled: false` to not rely on the  `cors_plug` configuration config key.
THis will allow us to add more plugs in the future, in the web I see already this possibility since we have at least two plugs to conditionally enable.

Reply:
@fabriziosestito This has the inconvenience that I would need to implement a new Unplug predicate function, as the current `Unplug.Predicates.AppConfigEquals` only works with 1 children values (like `:wanda, :enabled`).
Not a big deal, I could do it, but I want to share the limitation.